src/lib/storage/segment_accessor.cpp

```diff
@@ -13,7 +11,7 @@ std::unique_ptr<AbstractSegmentAccessor<T>> CreateSegmentAccessor<T>::create(
     using SegmentType = std::decay_t<decltype(typed_segment)>;
     if constexpr (std::is_same_v<SegmentType, ReferenceSegment>) {
       const auto& pos_list = *typed_segment.pos_list();
-      if (pos_list.references_single_chunk() && pos_list.size() > 0) {
+      if (pos_list.references_single_chunk() && !pos_list.empty()) {
```

Comment:
Just curious: did clang-tidy bring that up?

Reply:
Yepp